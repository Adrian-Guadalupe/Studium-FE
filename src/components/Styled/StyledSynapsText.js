import React from 'react';
import styled from 'styled-components';
import Text from '../../images/synapsText.png';

/*eslint no-unused-vars:0*/
const StyledSynapsText = props => {
  return (
    <SynapsText>
      <img src={Text} />
    </SynapsText>
  );
};

StyledSynapsText.propTypes = {};

const SynapsText = styled.div`
  font-family: Source Sans Pro;
  font-style: normal;
  font-weight: bold;
  font-size: 85px;
  line-height: 104.9%;
  /* or 128px */
  color: #231f20;
`;

export default StyledSynapsText;
