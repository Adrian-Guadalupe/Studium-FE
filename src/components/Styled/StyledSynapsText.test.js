import React from 'react';
import renderer from 'react-test-renderer';

import StyledSynapsText from './StyledSynapsText';



test('StyledSynapsText render correctly', () => {
    const tree = renderer.create(<StyledSynapsText />).toJSON();
    expect(tree).toMatchSnapshot();
});