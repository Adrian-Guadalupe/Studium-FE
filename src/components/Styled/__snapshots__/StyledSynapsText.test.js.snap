// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`StyledSynapsText render correctly 1`] = `
.c0 {
  font-family: Source Sans Pro;
  font-style: normal;
  font-weight: bold;
  font-size: 85px;
  line-height: 104.9%;
  color: #231F20;
}

<div
  className="c0"
>
  <img
    src="synapsText.png"
  />
</div>
`;
